pkgs/tools/networking/par2cmdline-turbo/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, stdenv, fetchFromGitHub, autoreconfHook }:
+
+stdenv.mkDerivation rec {
+  pname = "par2cmdline-turbo";
+  version = "1.1.0";
+
+  src = fetchFromGitHub {
+    owner = "animetosho";
+    repo = "par2cmdline-turbo";
+    rev = "v${version}";
+    sha256 = "sha256-+p6EeWn+tgZFRWSK4feQ4TIXJP+2iIcu9Ng0T6JNRCE=";
```

```suggestion
    hash = "sha256-+p6EeWn+tgZFRWSK4feQ4TIXJP+2iIcu9Ng0T6JNRCE=";
```
The hash string already describes the algorithm

---

Fixed